### PR TITLE
Adding monitoring dashboard artifacts

### PR DESCRIPTION
**Pull request by Thanu**
This p/r includes monitoring dashboard event streams, event store, event receiver and spark scripts artifacts.

### CODE REVIEW

File: components/org.wso2.ppaas.manager.console/console/controllers/menu/menu.json [hunk 1]
```diff
@@ -202,6 +202,19 @@
...
+            "context": "/",
+            "title": "Health Statistics",
+            "icon": "",
+            "block-color":"#f1c40f",
+            "permissionPaths": [
+                "/permission",
+                "/permission/admin"
+            ],
+            "description": "Monitor health statistics of clusters and members here."
+        },
```
Comment (R-Rajkumar) [inline components/org.wso2.ppaas.manager.console/console/controllers/menu/menu.json]: To be consistent, please remove the "here." from the description.

File: extensions/das/artifacts/monitoring-dashboard/capps/ppaas-monitoring-service/Dashboard/ppaas-monitoring-dashboard.json [hunk 2]
```diff
@@ -0,0 +1,274 @@
+{
```
Comment (R-Rajkumar) [inline extensions/das/artifacts/monitoring-dashboard/capps/ppaas-monitoring-service/Dashboard/ppaas-monitoring-dashboard.json]: Not sure whether we need a license header to this file?

File: components/org.wso2.ppaas.manager.console/console/controllers/menu/menu.json (hunk 1 above)
Comment (Thanu) [inline components/org.wso2.ppaas.manager.console/console/controllers/menu/menu.json]: Fixed in commit 0c3d20caf2808d35d95f886687cce0e0bf7248d0.

File: components/org.wso2.ppaas.manager.console/console/controllers/menu/menu.json [hunk 3]
```diff
@@ -202,6 +202,19 @@
             "description": "Manage applications."
         },
         {
+            "link": "https://localhost:9444/portal/dashboards/ppaas-monitoring-dashboard",
```
Comment (swgkg) [inline components/org.wso2.ppaas.manager.console/console/controllers/menu/menu.json]: Link is hard corded. It is better if you add this dashboard url as configurable

Comment (Thanu) [inline components/org.wso2.ppaas.manager.console/console/controllers/menu/menu.json]: Fixed in commit 90ea0e470d87b3ba88b9a90cb6038fa6b35fc87e.

**Pull request merged**